Models/Interior/Control-lock/Control-lock.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0"?>
+
+<PropertyList>
+
+    <path>Control-lock.ac</path>
+    <nopreview/>
+
+    <animation>
+        <type>select</type>
+        <object-name>ControlLock-yoke</object-name>
+        <condition>
+            <property>sim/model/c172p/cockpit/control-lock-placed</property>
+        </condition>
+    </animation>
+
+    <animation>
+        <type>select</type>
+        <object-name>ControlLock-pocket</object-name>
+        <condition>
+            <not>
+                <property>sim/model/c172p/cockpit/control-lock-placed</property>
+            </not>
+        </condition>
+    </animation>
+
+    <animation>
+        <type>pick</type>
+        <object-name>ControlLock-yoke</object-name>
+        <visible>true</visible>
+        <action>
+            <button>0</button>
+            <repeatable>false</repeatable>
+            <binding>
+                <command>property-assign</command>
+                <property>sim/model/c172p/cockpit/control-lock-placed</property>
+                <value>false</value>
+            </binding>
+        </action>
+        <hovered>
+            <binding>
+                <command>set-tooltip</command>
+                <tooltip-id>remove-control-lock</tooltip-id>
+                <label>Remove control lock</label>
+            </binding>
+        </hovered>
+    </animation>
+
+    <animation>
+        <type>pick</type>
+        <object-name>ControlLock-pocket</object-name>
+        <visible>true</visible>
+        <action>
+            <button>0</button>
+            <repeatable>false</repeatable>
+            <binding>
+                <condition>
+                    <greater-than>
+                        <property>sim/model/c172p/cockpit/yoke-aileron</property>
+                        <value>-0.05</value>
+                    </greater-than>
+                    <less-than>
+                        <property>sim/model/c172p/cockpit/yoke-aileron</property>
+                        <value>0.05</value>
+                    </less-than>
+                    <greater-than>
+                        <property>sim/model/c172p/cockpit/yoke-elevator</property>
+                        <value>-0.05</value>
+                    </greater-than>
+                    <less-than>
+                        <property>sim/model/c172p/cockpit/yoke-elevator</property>
+                        <value>0.05</value>
+                    </less-than>
```

Comment:
Perhaps these conditions above are not necessary. I get that what you implement might be more "realistic" from a certain point of view, but the problem is that it might look cryptic to the user: "hmm, why the lock can't be placed right now but was placed before?" Some people fly with mouse or keyboard and might not centralize the yoke back. IMO we should remove this